client/asset/dcr/dcr.go

```diff
@@ -555,14 +554,6 @@ func (dcr *ExchangeWallet) Connect(ctx context.Context) (*sync.WaitGroup, error)
 		}
 	}()
 
-	curnet, err := dcr.wallet.Network(ctx)
-	if err != nil {
-		return nil, fmt.Errorf("unable to fetch wallet network: %w", err)
-	}
-	if curnet != dcr.chainParams.Net {
-		return nil, fmt.Errorf("unexpected wallet network %s, expected %s", curnet, dcr.chainParams.Net)
```

Comment:
I thought it would be safer to make this check from the `ExchangeWallet` but I suppose since the `Network` method is only used here, maybe we can do without it and assume `Wallet` impls will do the right thing?

Reply:
Whatever really.